packages/client/src/views/Settings/items/Name.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as React from 'react'
+import { ListViewItemSimplified } from '@opencrvs/components/lib/interface'
+import { useIntl } from 'react-intl'
+import { userMessages, buttonMessages } from '@client/i18n/messages'
+import {
+  LabelContainer,
+  ValueContainer,
+  DynamicHeightLinkButton
+} from '@client/views/Settings/items/utils'
+import { useSelector } from 'react-redux'
+import { IStoreState } from '@client/store'
+import { getDefaultLanguage } from '@client/i18n/utils'
+import { GQLHumanName } from '@opencrvs/gateway/src/graphql/schema'
+
+export function Name() {
+  const intl = useIntl()
+  const englishName = useSelector<IStoreState, string>((state) => {
+    const { userDetails } = state.profile
+    const nameObj = userDetails?.name?.find(
+      (storedName: GQLHumanName | null) => {
+        const name = storedName as GQLHumanName
+        return name.use === getDefaultLanguage()
+      }
+    ) as GQLHumanName
+
+    return nameObj
+      ? `${String(nameObj.firstNames)} ${String(nameObj.familyName)}`
+      : ''
```

Comment:
There is a `getUserName` function in `userUtils.ts` that can probably be used here instead.

Reply:
Done in https://github.com/opencrvs/opencrvs-core/pull/3647/commits/4d5fcbd4d4a3809250db0d3e9e9c2b3d8741a424